src/pkgs/nix-directory.nix

```diff
@@ -34,9 +21,11 @@ in
 buildPkgs.stdenv.mkDerivation {
   name = "nixDirectory";
 
+  # FIXME: find a source where sha256 never changes, nixInstallerSha256
```

Comment:
Could you elaborate on this one? Why do you think it changes? Why not just add constant hashes per arch?

Reply:
When I think about it, we could probably hardcode the sha256 per arch. I was just thinking that this were rolling releases like the nix channel tar balls, but this one will probably never get updated.. My fault! I can fix it in the next days or you can do it, it's up to you. Thanks for pointing this out :)